README.md

```diff
@@ -1,2 +1,3 @@
 #Data Structures basic commands
-Add queue implimentation
\ No newline at end of file
+To contribute you may simply add DST codes in here:
+For examlple,Add queue implimentation.
```

Comment:
Please don't spam PRs only to increase Hacktoberfest count. Especially if the "typo fixes" contain typos themselves